tools/dashboard-plugin-template/package.json

```diff
@@ -65,6 +65,7 @@
         "@babel/preset-env": "^7.7.2",
         "@babel/preset-react": "^7.7.2",
         "@gooddata/eslint-config": "^2.1.0",
+        "@gooddata/plugin-toolkit": "^8.7.0-alpha.50",
```

Comment:
Why is this needed?

Reply:
to be part of the plugin project after bootstrap :)

the idea is that once user gets the plugin project (by using npx .. init) they have everything in that project to take things e2e. i'll eventually add additional scripts into the template so that user can do `npm run add-plugin` or `npm run use-plugin`.

Reply:
Oh right, makes sense!